### DISCUSSION TITLE
DietPi-Software | Baresip

### DISCUSSION THREAD

**Discussion by whitepj** [MichaIng/DietPi#5838] [Software Request]
A bit of a longshot, but... Would it be possible to add Baresip to the software list?

Using APT will allow the installation of v1.0.0-4, but this version goes back to 2020. The current release is 2.8.1
It is, as the name suggests, a very minimal headless SIP client -- perfect (I think) for DietPi.

Thanks.

**Comment by MichaIng** [comment]
Debian is generally aware of the upstream releaaes: https://tracker.debian.org/pkg/baresip
It additionally makes sense to open a bug report there and ask for an update. That way we also find out about the maintainer status.

There are other important tasks, but I'll have a look into how easy it difficult it is to compile and setup baresip when I find time. Any help on this, collecting information, testing build and/or setup, could speed it up.
